rust/src/treefile.rs

```diff
@@ -1490,5 +1490,25 @@ mod ffi {
             Box::from_raw(tf);
         }
     }
+
+    #[no_mangle]
+    pub extern "C" fn ror_treefile_get_files_to_remove(
+        tf: *mut Treefile,
+        package: *const libc::c_char,
+    ) -> *mut *mut libc::c_char {
+        let package = ffi_view_os_str(package);
+        let package = package.to_string_lossy().into_owned();
+        let mut files_to_remove: Vec<String> = Vec::new();
+        let tf = ref_from_raw_ptr(tf);
+        if let Some(ref packages) = tf.parsed.remove_from_packages {
+            for pkg in packages.iter() {
+                if pkg[0] == package {
+                    files_to_remove.extend_from_slice(&pkg[1..pkg.len()]);
+                    break;
```

Comment:
Hmm, don't think we should `break` here. See e.g. https://github.com/coreos/fedora-coreos-config/blob/a8f0228ae93e95946467f6d9f9f0a930bf795526/manifests/fedora-coreos-base.yaml#L43-L55 which as multiple entries for `systemd`.

Reply:
oops, you're right, didn't know that was possible.